src/bootloaders/shim-systemd.c

```diff
@@ -110,7 +111,7 @@ static char *shim_systemd_get_kernel_dst(const BootManager *manager) {
 
 static bool shim_systemd_install_kernel(const BootManager *manager, const Kernel *kernel) {
         fprintf(stderr, "Call %s\n", __func__);
-        return true;
+        return sd_class_install_kernel_impl(manager, kernel, shim_systemd_get_kernel_dst, NULL);
```

Comment:
Similar for this commit, it would be nice if the kernel install parts could be the same for sd_class and shim. I say this mostly as it seems easier to manage one rather than two locations and it doesn't really make sense to me to have a single distro with two kernels, one for secure boot and one for without.